src/FSharpx.Async/Async.fs

```diff
@@ -8,10 +8,14 @@ open System.Threading
 
 // ----------------------------------------------------------------------------
 
-[<AutoOpen>]
-module AsyncExtensions = 
-    type Microsoft.FSharp.Control.Async with 
+module AsyncOps =
```

Comment:
The only issue with making it internal is that then you can't inline the methods on `Async` that would call it. Premature optimization?